src/main/java/at/hannibal2/skyhanni/features/garden/visitor/VisitorListener.kt

```diff
@@ -33,20 +30,18 @@ import net.minecraftforge.client.event.GuiScreenEvent.KeyboardInputEvent
 import net.minecraftforge.event.entity.player.ItemTooltipEvent
 import net.minecraftforge.fml.common.eventhandler.EventPriority
 import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
-import org.lwjgl.input.Keyboard
 
 private val config get() = VisitorAPI.config
 
 class VisitorListener {
 
-    private var lastClickedNpc = 0
     private val logger = LorenzLogger("garden/visitors/listener")
 
     companion object {
 
-        private val VISITOR_INFO_ITEM_SLOT = 13
-        private val VISITOR_ACCEPT_ITEM_SLOT = 29
-        private val VISITOR_REFUSE_ITEM_SLOT = 33
+        private const val INFO_SLOT = 13
+        private const val ACCEPT_SLOT = 29
+        private const val REFUSE_SLOT = 33
```

Comment:
I should probably remove REFUSE_SLOT

Also whats the point of having them in companion object?